tendrl/commons/objects/__init__.py

```diff
@@ -169,8 +169,10 @@ def load_all(self):
                         return None
         ins = []
         for item in etcd_resp.leaves:
-            self.value = item.key
-            ins.append(self.load())
+            if item.key.strip("/") != value.strip("/"):
```

Comment:
what is this checking of key and value part? Can you add an example here?

Reply:
i will add